plone/recipe/codeanalysis/__init__.py

```diff
@@ -77,6 +79,9 @@ def __init__(self, buildout, name, options):
         # CSS Lint
         self.options.setdefault('csslint', 'False')
         self.options.setdefault('csslint-bin', 'csslint')
+        # check-manifest
+        self.options.setdefault('check-manifest', 'True')
```

Comment:
We were talking about the default value, which should be false. I also started to wondering if p.r.codeanalysis is the right place for such a check. The manifest check is just important for releases. Shouldn't we rather do this check as part of the release process (e.g. add it to zest.releaser or plone.releaser)?